set.mm

```diff
@@ -505888,6 +505901,19 @@ its graph has a given second element (that is, function value).
 $)
 
 
+  $( Alternate value of the signum of a real number.  (Contributed by Thierry
```

Comment:
"Alternative value"?

Reply:
> Isn't it less an alternative _value_ (since the signum remains what it is) but alternative _formula_ to calculate it?

Yes: the new version https://github.com/metamath/set.mm/pull/5093/commits/8f9f43eb051b503c9663c65ef2b2a00a231ec535 addresses this and completely removes both "alternate" and "alternative" in the wording!